src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1559,10 +1559,10 @@ public function getParameterBag(): ParameterBagInterface
         if ($dynamicPhp) {
             $loadedDynamicParameters = $this->exportParameters(array_combine(array_keys($dynamicPhp), array_fill(0, \count($dynamicPhp), false)), '', 8);
             $getDynamicParameter = <<<'EOF'
-        switch ($name) {
+        $value = match ($name) {
```

Comment:
@nicolas-grekas This change is not really minor I guess, should I submit it as a feature in another PR?

Reply:
I'd keep it here, it's the very same topic